### ISSUE TITLE
Why use torch.cat([tenEncone[intLevel], tenMetricone] as softsplat input tenIn

### ISSUE THREAD

**Issue by Candy-Crusher** [sniklaus/softmax-splatting#63]
<img width="748" alt="1729073895277" src="https://github.com/user-attachments/assets/7470c431-f164-4bb7-ad35-5ba646381310">

Hi dear authors,

Thanks for the amazing work. I have a confusion about softsplat input. According to my understanding, the tenIn shoud be image features solely. But in the code, I notice that you also concate tenMetricone. May I ask the reason behind it?

**Comment by sniklaus** [comment]
Thank you for the kind words! It has been a while, but if I remember correctly then providing this metric to the synthesis network slightly improves the interpolation quality. So it doesn't have anything to do with the softmax splattig itself, just with providing more information to the synthesis network. Hope that helps!
